demo-java/app/build.gradle

```diff
@@ -1,6 +1,9 @@
 import org.apache.tools.ant.filters.ConcatFilter
 
-apply plugin: 'com.android.application'
+plugins {
+    id 'com.android.application'
+    id 'com.google.secrets_gradle_plugin' version '0.6'
```

Comment:
Thanks for adding this! I think it makes sense to integrate your change instead of #135 - will close my PR.

Reply:
I tried to grab all the changes from your PR for merge compatibility :)